### DISCUSSION TITLE
VS Code Typst YAML linting error

### DISCUSSION THREAD

**Discussion by riziles** [quarto-dev/quarto-cli#9504] [Q&A] [answered]
### Description

If I use Typst as my format, VS Code throws a linting error (`Field "format" has value typst, which must instead be 'asciidoc'`). Is there any way to make it go away? See example below.

```qmd
---
title: Quarto-Typst Demo
format:
  typst
---

Lorem ipsum
```

Using VS Code Extension 1.111.0 on Windows.

**Comment by mcanouil** [comment]
Which version of Quarto CLI are you using?

**Reply by riziles** [reply]
1.4.550

**Reply by mcanouil** [reply]
I cannot reproduce
<img width="686" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/28a6c4ae-16e9-450d-840c-ff268b4dc340">

Could you expand a bit more on how to reproduce?

**Comment by riziles** [comment, marked as answer]
> I cannot reproduce

Ahh! Thank you! This was actually very helpful. I didn't realize the problem was me. I had two conflicting versions of Quarto: an older one installed with the official Quarto installer and a newer one installed with mamba. VS Code preview function was using the mamba one, but the linter was using the older one. Marking this as answer in case anybody else wants up here.

**Reply by mcanouil** [reply]
Glad you figured it out.
